object-browser.md

```diff
@@ -0,0 +1,68 @@
+```hbs
+{{frost-object-browser-lts
+  items=sortedItems
+  header=(component 'frost-info-bar'
+    icon=(component 'frost-icon'
+      icon='list-medium'
+      pack='frost'
+    )
+    title=(component 'text-box'
```

Comment:
What's a `text-box` component?  Why not just a string instead of a component helper?